smart-contracts/contracts/lst-dex-adapter-osmosis/src/msg.rs

```diff
@@ -0,0 +1,47 @@
+use crate::contract::DexAdapter;
+
+use abstract_app::objects::PoolAddress;
+use cosmwasm_schema::QueryResponses;
+use cosmwasm_std::Decimal;
+use cw_asset::AssetInfo;
+
+abstract_app::app_msg_types!(DexAdapter, DexAdapterExecuteMsg, DexAdapterQueryMsg);
+
+#[cosmwasm_schema::cw_serde]
+pub struct DexAdapterInstantiateMsg {
```

Comment:
yes there is a reason, because of abstractSDK. The code in line 8 generates the ExecuteMsg and QueryMsg for the abstractSDK wrapper of the contract. Therefore, I have to choose between two suboptimal solutions:
* longer names for messages as I did
* move the macro invocation to another file
First solution gives longer names, second moves messages out of msg.rs. 
I don't care about which solution to choose at all, just went with the approach the abstract guys were using in their examples

Reply:
fair, lets keep with the examples then